examples/rosetta/bioinformatics - base count.art

```diff
@@ -19,12 +19,12 @@ prettyPrint: function [in][
         print split.every:10 line
 
         loop split line 'ch [
```

Comment:
tally

Reply:
That was... brutal. Practically half of the code is gone! 🤣